licensing/src/main/java/io/crate/license/DecryptedLicenseData.java

```diff
@@ -53,23 +64,68 @@ String issuedTo() {
         return issuedTo;
     }
 
+    String signature() {
+        return (signature.isPresent() ? signature.get() : "");
+    }
+
+    void setSignature(String signature) {
```

Comment:
Why do we need a setter for this?

Reply:
It is used by the service [to bundle the newly created signature to the license](https://github.com/crate/crate/pull/7758/files#diff-4b272f5578419e39b7c0a1db3b7ca030R195).